src/plotly.js

```diff
@@ -37,6 +37,7 @@ exports.Shapes = require('./components/shapes');
 exports.Legend = require('./components/legend');
 exports.Images = require('./components/images');
 exports.UpdateMenus = require('./components/updatemenus');
+exports.Slider = require('./components/slider');
```

Comment:
But you see, `RangeSlider` is consistent with `layout.xaxis.rangeslider` i.e. the option container name (even though we might want to support multiple range sliders in the future).